soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -35,6 +35,12 @@ enum MatchSituation {
 
 enum Positions { Goalie, Defense, Offense };
 
+// These values are explicitly declared because they are the ints that are published to
+// strategy/positions i.e. the same values as strategy::Positions
+namespace OverridePosition {
+enum OverridePosition { Goalie = 0, Defense = 1, Offense = 2, None = 3 };
```

Comment:
@p-nayak11 I think this would be a better enum for your led PR

also @sid-parikh why separate namespace? I think leaving it under strategy namespace is fine here

Reply:
separate namespace otherwise you get a compiler error for redefining `Strategy::Goalie`, `Strategy::Defense`, and `Strategy::Offense`. I needed a None hence the new enum